README.md

```diff
@@ -26,20 +26,9 @@ brew tap GoogleContainerTools/kpt https://github.com/GoogleContainerTools/kpt.gi
 brew install kpt
 ```
 
-### Install from binaries
+### Download latest release
 
-| Platform
-| ------------------------
-| [Linux (x64)][linux]
-| [macOS (x64)][darwin]
-| [Windows (x64)][windows]
-
-```sh
-# For linux/mac
-chmod +x kpt
```

Comment:
These instructions would be missing. Can we add this ?

Reply:
This shouldn't be needed. The binary in the `tar.gz` file already has the execution bit set.